client/tracing/src/logging/mod.rs

```diff
@@ -125,21 +125,27 @@ where
 	}
 
 	if directives != "" {
-		// We're not sure if log or tracing is available at this moment, so silently ignore the
-		// parse error.
 		env_filter = parse_user_directives(env_filter, directives)?;
 	}
 
+	if let Some(profiling_targets) = profiling_targets {
```

Comment:
I feel like I am having a déjà vu. Didn't this code use to be pretty much like this? (included the forced `sc_tracing=trace` directive).